src/main/java/org/sonar/plugins/scm/perforce/PerforceConfiguration.java

```diff
@@ -137,4 +148,9 @@ public String clientName() {
     return settings.getString(CLIENT_PROP_KEY);
   }
 
+  @CheckForNull
```

Comment:
Yep, you're right.  Originally was a String but missed that when I changed it to an int.